lang/cs/Org.Apache.REEF.Network/Naming/NameServer.cs

```diff
@@ -50,8 +51,13 @@ public class NameServer : INameServer
         /// Create a new NameServer to run on the specified port.
         /// </summary>
         /// <param name="port">The port to listen for incoming connections on.</param>
+        /// <param name="tcpPortProvider">If port is 0, this interface provides 
+        /// a port range to try.
+        /// </param>
         [Inject]
-        public NameServer([Parameter(typeof(NamingConfigurationOptions.NameServerPort))] int port)
+        internal NameServer(
```

Comment:
We probably have to keep this `public` as per our deprecation promise. We can mark it as `obsolete`, though. And make it `internal` right after the next release.